ext/mbstring/libmbfl/filters/mbfilter_cp1252.c

```diff
@@ -126,17 +128,10 @@ int mbfl_filt_conv_cp1252_wchar(int c, mbfl_convert_filter *filter)
 	return c;
 }
 
-/* We only distinguish the MS extensions to ISO-8859-1.
- * Actually, this is pretty much a NO-OP, since the identification
- * system doesn't allow us to discriminate between a positive match,
- * a possible match and a definite non-match.
- * The problem here is that cp1252 looks like SJIS for certain chars.
- * */
 static int mbfl_filt_ident_cp1252(int c, mbfl_identify_filter *filter)
 {
-	if (c >= 0x80 && c < 0xa0)
-		filter->flag = 0;
-	else
-		filter->flag = 1; /* not it */
+	if (c >= 0x80 && c < 0xa0 && cp1252_ucs_table[c - 0x80] == 0xFFFD) {
```

Comment:
We now have two different conversions: The iso8859 tables use 0x0 to indicate an unmapped character, while here we use 0xfffd to indicate the same. I think we should canonicalize to one of these.

Reply:
Good point. Done.